### ISSUE TITLE
Swift Package Manager via XcodeUI without RXSwift dependencies

### ISSUE THREAD

**Issue by cl0ter** [Moya/Moya#2210]
I'm integrated Moya+Combine with Swift Package Manager (XcodeUI). When dialog window with package type was shown I choose only MoyaCombine. But RxSwift was added as dependency. 

Could I use Moya+Combine only? Without RxSwift dependency. 

Environment: iOS application project

**Comment by cl0ter** [comment]
@ashfurrow I apologize for disturbing. Is there an opportunity to consider this?

**Comment by ashfurrow** [comment]
Hi, that sounds reasonable. I confess that I don't use SPM much so I'm not sure why this is happening or how to fix it. But supporting Moya and Combine only seems like a good idea 👍

**Comment by ajjumehra** [comment]
We are also using the Moya in our project. and we are using reactiveswift instead of Rxswift but, Moya is pulling Rxswift too. I don't think there is a benefit on that keeping two reactive lib

**Comment by markst** [comment]
Same goes for `RxMoya` seems to include `ReactiveSwift`.

**Comment by MnkGitBox** [comment]
Yes. Same here, RxMoya installs when only selected Combine.  It is not a good idea to put unwanted libraries in the project I think.

**Comment by lordcodes** [comment]
I think it is that when resolving a package in Xcode it downloads all of the dependency packages and shows them in the project packages area. However, only the ones you include will actually be used within your targets.
It is slightly irritating as it means you will be downloading stuff you then never use. I have seen it where a package includes SwiftLint in its dependencies list and then uses it on another internal target but not in any of the output products. This included a huge number of dependencies into the Xcode UI.
I have a feeling the only way to "fix" this would be for RxMoya and ReactiveMoya to be separate repositories.

**Comment by GeoSD** [comment]
And the same problem here. 
I'm using Moya as is, without Combine or RxSwift of ReactiveSwift. I've just finished removing RxSwift from project and really don't want to see it in dependencies. That because any new developer in project may start using RxSwift again.
Or anyway unused code should not be in project.
